cmd/adapter/config.go

```diff
@@ -0,0 +1,272 @@
+/*
+Copyright 2019 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/logrange/logrange/pkg/forwarder"
+	"github.com/logrange/logrange/pkg/utils"
+	"github.com/logrange/range/pkg/transport"
+	"io/ioutil"
+)
+
+type (
+	gravity struct {
+		ApiListenAddr string
+		Kubernetes    *kube
+	}
+
+	kube struct {
+		Namespace              string
+		ForwarderConfigMapName string
+	}
+
+	// cronQuery describe a query, which should be called periodically
+	cronQuery struct {
+		// Query contains LQL which should be called periodically
+		Query string
+		// PeriodSec defines an interval in seconds to execute the Query
+		PeriodSec int
```

Comment:
Why is this not a `time.Duration`? You wouldn't have to convert it then (I saw it in a few places).

Reply:
I think it's fine as seconds - all conversions I've seen are superfluous and can be removed.

Reply:
mmm, somehow we used to have it seconds, so leaving it as is, until strong disagreement...